src/store/auth/actions.ts

```diff
@@ -1,31 +1,55 @@
 import {
+  AUTH_AUTHENTICATE_USER,
+  AUTH_SESSION_EXPIRED,
   AUTH_SIGN_IN_SUCCESS,
   AUTH_SIGN_OUT,
   AUTH_SIGN_UP_SUCCESS,
   AuthActionTypes,
+  AuthHeaders,
   SignInData,
   SignUpData
 } from './types';
 import { CmwlThunkAction } from '../index';
+import axiosCromwell from '../../axiosCromwell';
+import { AxiosResponse } from "axios";
 
-const signInSuccess = (): AuthActionTypes => ({
-  type: AUTH_SIGN_IN_SUCCESS
+const signInSuccess = (authHeaders: AuthHeaders): AuthActionTypes => ({
+  type: AUTH_SIGN_IN_SUCCESS,
+  authHeaders: authHeaders
 });
 
-const signUpSuccess = (): AuthActionTypes => ({
-  type: AUTH_SIGN_UP_SUCCESS
+const signUpSuccess = (authHeaders: AuthHeaders): AuthActionTypes => ({
+  type: AUTH_SIGN_UP_SUCCESS,
+  authHeaders: authHeaders
 });
 
 export const signIn = (signInData: SignInData): CmwlThunkAction => (dispatch => {
-  dispatch(signInSuccess());
-  // ToDo: Add axios async call
+  axiosCromwell.post('/auth/signIn', signInData).then(response => {
+    const authHeaders: AuthHeaders = getAuthHeaders(response);
+    dispatch(signInSuccess(authHeaders));
+  }).catch(error => console.log(error))
 });
 
 export const signUp = (signUpData: SignUpData): CmwlThunkAction => (dispatch => {
-  dispatch(signUpSuccess());
-  // ToDo: Add axios async call
+  axiosCromwell.post('/auth/signUp', signUpData).then(response => {
+    const authHeaders: AuthHeaders = getAuthHeaders(response);
```

Comment:
I'd suggest moving `: AuthHeaders` to the definition of `getAuthHeaders` function, and omit it here. It'd be more clean.

Reply:
Exactly! :+1: 
Done